scripts/release/analyze.ts

```diff
@@ -0,0 +1,204 @@
+import type { Commit, PackageToRelease } from "./types"
+import type { PackageJson } from "type-fest"
+
+import path from "path"
+import { execSync } from "child_process"
+// @ts-ignore
+import jsonfile from "jsonfile"
+import semver from "semver"
+import parseCommit from "@commitlint/parse"
+// @ts-ignore
+import gitLog from "git-log-parser"
+// @ts-ignore
```

Comment:
Trivial: you can add `@types/stream-to-array` and `@types/jsonfile` and remove `// @ts-ignore`